app-arch/par2cmdline-turbo/par2cmdline-turbo-1.0.1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools toolchain-funcs
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/animetosho/par2cmdline-turbo"
+else
+	SRC_URI="https://github.com/animetosho/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86 ~amd64-linux"
```

Comment:
Let's drop everything but ~amd64 please pending rekeywording. I assume these are inherited from par2cmdline but I wouldn't be surprised if the SIMD stuff breaks it.